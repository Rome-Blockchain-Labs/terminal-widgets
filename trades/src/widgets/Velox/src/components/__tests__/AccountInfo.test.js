import { shallow } from 'enzyme';

import { AccountInfo } from '../AccountInfo';

describe('AccountInfo component render', () => {
  it('AccountInfo snapshot added', () => {
    const component = shallow(<AccountInfo />);
    expect(component).toMatchSnapshot();
  });
});
