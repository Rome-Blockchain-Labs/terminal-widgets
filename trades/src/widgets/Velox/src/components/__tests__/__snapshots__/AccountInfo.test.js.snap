// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccountInfo component render AccountInfo snapshot added 1`] = `
<div>
  <VeloxLinePanel
    className="auth__row"
  >
    <VeloxBlueLabel
      className="input-descr"
      prefix="auth"
    >
      ChainId: 
    </VeloxBlueLabel>
  </VeloxLinePanel>
  <VeloxLinePanel
    className="auth__row"
  >
    <VeloxBlueLabel
      className="input-descr"
      prefix="auth"
    >
      Account: 
    </VeloxBlueLabel>
  </VeloxLinePanel>
  <VeloxLinePanel
    className="auth__row"
  >
    <VeloxBlueLabel
      className="input-descr"
      prefix="auth"
    >
      <Balance />
    </VeloxBlueLabel>
  </VeloxLinePanel>
</div>
`;
